src/backend/distributed/sql/udfs/citus_finish_pg_upgrade/latest.sql

```diff
@@ -52,7 +52,7 @@ BEGIN
     FOR table_name IN SELECT logicalrelid FROM pg_catalog.pg_dist_partition
     LOOP
         trigger_name := 'truncate_trigger_' || table_name::oid;
-        command := 'create trigger ' || trigger_name || ' after truncate on ' || table_name || ' execute procedure pg_catalog.citus_truncate_trigger()';
```

Comment:
@marcocitus This change brings triggers created by the finsh_pg_upgrade UDF in line with the ones created by `create_distributed_table`: https://github.com/citusdata/citus/blob/492d1b2cbad626a5e3cfee01e972dc922f18dcfb/src/backend/distributed/commands/create_distributed_table.c#L1163

I'm not sure which one is better though, or if it really matters. I could also change the C code to create an AFTER trigger instead if that's what we want.

Reply:
Be careful about changing these files. We can no longer change 9.0.

Reply:
I don't know whether it matters, that's worth figuring out (e.g. do tests still pass if we do AFTER triggers in C, does truncate MX concurrently with other commands still behave as expected)

Reply:
Changed the C code to create an AFTER trigger and replace all old truncate triggers with an AFTER trigger.